include/revng/PipelineC/ForwardDeclarations.h

```diff
@@ -18,6 +26,6 @@ typedef const pipeline::Target rp_target;
 typedef const pipeline::TargetsList rp_targets_list;
 typedef const pipeline::Step::AnalysisValueType rp_analysis;
 typedef const pipeline::DiffMap rp_diff_map;
-typedef revng::ErrorList rp_error_list;
+typedef std::vector<rp_error> rp_error_list;
```

Comment:
In the [design pad](https://pad.rev.ng/vK82o4YtRXKv2A_ctMjHKg#), `rp_error_list` also exposed `getLocationTypeName` and `getErrorTypeName`. Can we do that?